python/audio2text/main.py

```diff
@@ -0,0 +1,71 @@
+
+import time
+import requests
+from scramjet.streams import Stream
+
+
+class AssemblyAi():
+
+    def __init__(self, token, file):
+        """Main constructor
+        Args:
+            token (str): Auth token for AssemblyAi
+            file (str): Path to file with song
+        """
```

Comment:
I don't know what documentation guidelines are in Scramjet. This docstring was only my proposition

Reply:
would `type-hints` instead solve the uniformity issues ?